inc/define.php

```diff
@@ -510,6 +513,11 @@
    'self-service' => ['tinymce']
 ];
 
+// push reservations libs to reservations itemtypes (they shoul in asset sector)
+foreach ($CFG_GLPI['reservation_types'] as $reservation_type) {
+   $CFG_GLPI['javascript']['assets'][strtolower($reservation_type)] = $reservations_libs;
```

Comment:
This may unvonluntary override libs definition of these assets.
```suggestion
   $defined_libs = $CFG_GLPI['javascript']['assets'][strtolower($reservation_type)] ?? [];
   $CFG_GLPI['javascript']['assets'][strtolower($reservation_type)] = array_merge($defined_libs, $reservations_libs);
```